docs/walkthroughs/test-dev-dependency.md

```diff
@@ -0,0 +1,113 @@
+# Working with Test and Development Dependency
+
+FOSSA CLI, by default, does not include the unused dependency in analysis. This is expected for scenarios in which, you have a dependency for development, testing, and other purposes - which ultimately are not used in your runtime. FOSSA CLI can infer dependency's inclusion for runtime or distribution by package manager's manifest file. When we are unable to infer if the dependency is only used in runtime, we choose the conservative approach and include them in our analysis reporting. 
+
+CLI's language support documentation (in [strategy's section](./../references/strategies/README.md)), includes whether an analysis of a particular project will by default unused dependency.
+
+## Example 
+
+For the following `package.json` file:
+
+```json
+{
+    "name": "example-project",
+    "dependencies": {
+        "pngjs": "6.0.0",
+        "once": "1.4.0"
+    },
+    "devDependencies": {
+        "mocha": "*"
+    }
+}
+```
+
+### When running `fossa analyze`:
+
+We will report the dependency tree as follows:
+
+```tree
+.
+ |-npm+once$1.4.0 (direct)
+ | \-npm+wrappy$1 (transitive)
```

Comment:
Where is `wrappy` in the `package.json`? You may want to make the example file a `package-lock.json` file. I don't think the jump to where the transitive deps are coming from will be immediately obvious to most people reading this doc. This also goes for the dependency tree towards the bottom with dev deps in it. I was confused at first about where all of the other deps came from.